src/cross_section/src/cross_section.cpp

```diff
@@ -291,7 +291,7 @@ std::shared_ptr<const PathImpl> CrossSection::GetPaths() const {
   if (transform_ == glm::mat3x2(1.0f)) {
     return paths_;
   }
-  paths_ = shared_paths(transform(paths_->paths_, transform_));
+  paths_ = shared_paths(::transform(paths_->paths_, transform_));
```

Comment:
What does `::` do?

Reply:
It search for the transform function defined in the anonymous namespace. By default it should search for the one in the current namespace, i.e. the transform function defined in par.
I found this error when I was experimenting with precompiled headers (minor compile time improvement)